### ISSUE TITLE
Add "command-D" keyboard shortcut to "Do you want to save?" dialog window in Mac client

### ISSUE THREAD

**Issue by steresi** [a85/POSTMan-Chrome-Extension#3316]
When closing a tab and seeing a "Do you want to save?" dialog box, I'm not able to dismiss the window using the keyboard in all cases on the Mac Postman app.

Can "command-D" please be assigned to the "Don't Save" button?

Thank you a million for this usability improvement!!

Screenshot:

<img width="507" alt="screen shot 2017-07-26 at 4 18 54 pm" src="https://user-images.githubusercontent.com/6819819/28641978-d22996c6-721e-11e7-9f9f-e9e8fb9fe6c2.png">

## App Details:
```
Postman for Mac
Version 5.1.2
OS X 16.7.0 / x64
```

## How to reproduce:

Open a Postman request in a Collection
Modify the request body
Try to close the tab
The dialog box will appear

**Comment by sdnts** [comment]
Hi @steresi, thanks for reporting this, we've made a fix for this, and it will be out with the next release (5.2)

**Comment by quintrino** [comment]
Hi, Did the change for this alter the default behaviour listed in #2110 ? I used to be able to use spacebar to auto close with saving, but now it seems the auto focus has shifted to save.

**Comment by sdnts** [comment]
@quintrino This fix isn't out yet. On the current latest version (5.1.3), I tried this, and the focus is on `Save`, and pressing spacebar will save. Are you not seeing this?

**Comment by quintrino** [comment]
@madebysid Thanks for getting back to me! :)

Currently when I use the spacebar **save** has focus so it goes to save, but previously **don't save** had focus as listed in #2110.

I'm looking for a way to close tabs without saving, but now that the focus has changed it's a bit more difficult. Just curious when the change happened.

**Comment by bhawna2109** [comment]
Hi @quintrino - will the solution mentioned here #2195 suit your workflow?

**Comment by quintrino** [comment]
@bhawna2109 reading through #2195 [#issuecomment-235804708](https://github.com/postmanlabs/postman-app-support/issues/2195#issuecomment-235804708) also mentions spacebar being the keyboard shortcut for Don't save when the save modal pops up, but that's been changed so that spacebar now goes to save instead.

**Comment by bhawna2109** [comment]
@quintrino that's right. This was done to prevent accidental loss of data.

**Comment by quintrino** [comment]
Ah okay thanks @bhawna2109, now that 5.2 is out, what's the fix that @madebysid is referring to? 

I've had a look through the keyboard shortcuts of 5.2 but can't see anything about how to dismiss a save box via keyboard.

**Comment by sdnts** [comment]
@quintrino There isn't a dedicated shortcut (yet), but you can use the `Tab` key to switch between the available buttons.
The fix I was referring to wasn't added in 5.2 because we want to solve this for good. We've decided to indeed add a dedicated shortcut for this. I'm adding this to our 5.3 roadmap. Thanks for your suggestions & patience :)

**Comment by bhawna2109** [comment]
Hi guys, ability to force close all tabs is now supported in our latest version [(V 5.3.0)](https://www.getpostman.com/apps#changelog). 🎉 Clicking on the three dots next to the '+' icon brings up a dropdown with options to manage the tabs you're currently working with.

The last option in the dropdown - 'Force close all tabs' - will close all the tabs without prompting to save each time - 
![image](https://user-images.githubusercontent.com/14976346/31327774-eca078a2-acee-11e7-8a7d-52a290bef376.png)

Additionally, you can use CMD/CTRL + ALT + W shortcut to force close the currently active tab. Holding down the ALT key and clicking on the 'x' icon also closes a tab without bring up the save modal.

**Comment by steresi** [comment]
Okay, I'm using Postman 5.3 and still hoping we will eventually get a keyboard shortcut for "Don't Save" when closing an individual tab. I believe in Mac dialog boxes, the default keyboard shortcut for each button was "command" plus "first letter of the button name." I don't know if that's still a Mac standard, but it's the behavior in Microsoft Word and BBEdit for instance.

**Comment by a85** [comment]
@steresi The shortcut is Cmd/Ctrl + Alt + W as @bhawna2109 mentioned in her comment.

**Comment by steresi** [comment]
I see! Thanks!

However, if you hit command-W to close the tab, it still brings up the "Do you want to save your changes?" dialog box, and there's no keyboard shortcut for the "Don't Save" button. You have to think ahead of time and hit command-option-W. That will close the tab without bringing up the dialog box.

That seems a bit like unusual behavior... I'm not familiar with a Mac app that gives you a keyboard shortcut for "close a document without saving"! (We are treating tabs here like documents.) Maybe I've just never looked. Mac apps usually bring up the "Do you want to save your changes?" dialog box, and then you can use command-D to select "Don't Save".

However the command-option-W behavior does let me close a tab without saving, so I will just get used to it!

**Comment by a85** [comment]
@steresi Good point. We'll add the shortcut to the button itself. Getting the actual behavior consistent across different platforms along with Postman's use cases is hard but we find this solution to work for everyone.

**Comment by DavidKahnt** [comment]
Is there any ongoing effort to implement this feature request? It would be nice to have it working "cross-platform", so you can also use your short keys on linux, windows and mac.

I am currently using Postman v7.2.2 and can't use short keys, or do I miss something?
